mst/grafana-mst.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Integrate Managed Service for TimescaleDB as a data source in Grafana
+excerpt: Integrate Managed Service for TimescaleDB as a data source in Grafana to visualize your data
+keywords: [Grafana, visualizations, analytics]
+---
+
+# Integrate Managed Service for TimescaleDB and Grafana
+
+You can integrate the services within the Managed Service for TimescaleDB platform
+to tranform into a true data cloud. Grafana ships with built-in Prometheus,
```

Comment:
This sentence doesn't feels like it doesn't really say anything. Can we say something more meaningful, like "You can use Managed Service for TimescaleDB with Grafana to visualize your data" or similar? Let's get to what's actually happening in this section.